src/ecephys_etl/modules/vcn_create_stimulus_table/README.md

```diff
@@ -0,0 +1,29 @@
+Stimulus Table
```

Comment:
why does vcn have a README, but not vbn?
could we just bump it up one level and generalize?

Reply:
I think there should be an individual README per module and a README for the module directory. Just because in the future there will be more modules and composite README containing info for every module would be pretty hard to search/read.